todo_app/app.py

```diff
@@ -1,16 +1,20 @@
 from flask import Flask, redirect, render_template, request
 from todo_app.flask_config import Config
 from todo_app.data import session_items
```

Comment:
I think these session_items (and the Config above it) are now unused? If so we can remove these imports and the files themselves